bitaxe_hashrate_benchmark.py

```
@@ -4,17 +4,25 @@
 import signal
 import sys
 import argparse
+import threading
```

Ruff (0.12.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)